core/src/main/java/com/megaease/easeagent/core/Register.java

```diff
@@ -43,7 +44,7 @@
     }
 
     void apply(String adviceClassName, ClassLoader external) {
-        if (!applied.add(adviceClassName + external.getClass().getName())) return;
+        if (!applied.add(adviceClassName + "-" + System.identityHashCode(external))) return;
```

Comment:
`Object.hashCode`、`Objects.hashCode `、`System.identityHashCode` cannot  guarantee the uniqueness of the object.
The object should be used as the key:
```
class ComboKey{
    String adviceClassName;
    ClassLoader classLoader;
}
```